core/client/fs/src/main/java/alluxio/client/file/AlluxioEvent.java

```diff
@@ -0,0 +1,78 @@
+package alluxio.client.file;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Defines and Manages well-known Alluxio events.
+ *
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ * !!! - Keep increasing the event-id for new events. !!!
+ * !!! - Don't reassign event ids.                    !!!
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ */
+public enum AlluxioEvent {
+  // Alluxio master events.
+  Master_Process_Starting(1, EventType.INFO, "Master process starting."),
```

Comment:
I feel like these should either be camelcase, or all caps with underscores.

Reply:
I tried all caps and it looked intrusive. 
I will try camel casing.

Reply:
camel casing is better.